snuba/settings/__init__.py

```diff
@@ -149,6 +149,9 @@
 # Metric Alerts Subscription Options
 ENABLE_SESSIONS_SUBSCRIPTIONS = os.environ.get("ENABLE_SESSIONS_SUBSCRIPTIONS", False)
 
+# The maximum number of spans to store in the array columns for each transaction.
+MAX_SPANS_PER_TRANSACTION = 2000
```

Comment:
why not just make this 1000?

Reply:
2000 was chosen after some discussion with @evanh